tb_plugin/README.md

```diff
@@ -45,3 +45,98 @@
+
+Device Self Duration: The time spent on GPU(maybe not GPU fully utilized during this time range), not including this operator’s sub-functions.
+
+Device Total Duration: The time spent on GPU(maybe not GPU fully utilized during this time range), including this operator’s sub-functions.
+
+Host Self Duration: The time spent on Host(maybe not CPU busy during this time range), not including this operator’s sub-functions.
+
+Host Total Duration: The time spent on Host(maybe not CPU busy during this time range), including this operator’s sub-functions.
+
+The top 4 pie charts are visualizations of the above 4 columns. 
+
+“Group By” could choose between “Operator” and “Operator + Input Shape”. The “Input Shape” is each operator’s input argument list’s tensor shapes, “[]” is scalar type. For example, “[[32, 256, 14, 14], [1024, 256, 1, 1], [], [], [], [], [], [], []]” means this operator has 9 input arguments, 1st is a tensor of size 32\*256\*14\*14, 2nd is a tensor of size 1024\*256\*1\*1, the following 7 ones are scalar variables.
+
+![Alt text](https://github.com/gaoteng-git/kineto/blob/readme/tb_plugin/tensorboard_plugin_torch_profiler/docs/images/operator_view_group_by_inputshape.PNG)
+
+* Kernel View
+
+This view shows all kernels’ time spent on GPU. It is got from kernel’s end time subtract kernel’s start time. It does not include cudaMemcpy or cudaMemset now.
+
+Note: This time just records a kernel's eplased time on GPU device. It does not mean GPU is fully busy during this time interval. That is, GPU occupancy may be less than 100% during this time interval. 
+
+![Alt text](https://github.com/gaoteng-git/kineto/blob/readme/tb_plugin/tensorboard_plugin_torch_profiler/docs/images/kernel_view.PNG)
+
+Total Duration: The summarization time of all calls of this kernel.
+
+Mean Duration: Total Duration/Calls.
+
+Max Duration: Max{Duration of all Calls}.
+
+Min Duration: Min{Duration of all Calls}.
+
+The top pie is a visualization of "Total Duration" column. 
+
+* Trace View
+
+![Alt text](https://github.com/gaoteng-git/kineto/blob/readme/tb_plugin/tensorboard_plugin_torch_profiler/docs/images/trace_view.PNG)
+
+The “thread 0” is the CPU thread that do “backward” of neural network.
+
+The “thread 1” is the main CPU thread, which mainly do data loading, forward of neural network, Optimizer.step.
+
+The “stream 7” is CUDA stream trace line, you can see all kernels here.
+
+We can see there are 6 “ProfilerStep” we captured.
+
+We cat scroll the bar to zoom into it.
+
+![Alt text](https://github.com/gaoteng-git/kineto/blob/readme/tb_plugin/tensorboard_plugin_torch_profiler/docs/images/trace_view_one_step.PNG)
+
+The “Optimizer.step#SGD.step”、”enumerate(DataLoader)#_SingleProcessDataLoaderIter.__next__” are high-level python side operations.
+
+![Alt text](https://github.com/gaoteng-git/kineto/blob/readme/tb_plugin/tensorboard_plugin_torch_profiler/docs/images/trace_view_launch.PNG)
```

Comment:
the images are on your repo.

Reply:
I'm also wondering about this. Let me update these links to formal repo after this PR is merged. Because the images are not uploaded to formal repo before this PR is merged.